opencti-platform/opencti-graphql/src/resolvers/attribute.js

```diff
@@ -1,9 +1,9 @@
-import { getRuntimeAttributeValues, getSchemaAttributeValues } from '../domain/attribute';
+import { getRuntimeAttributeValues, getSchemaAttributeNames } from '../domain/attribute';
 
 const attributeResolvers = {
   Query: {
     runtimeAttributes: (_, args, context) => getRuntimeAttributeValues(context, context.user, args),
-    schemaAttributes: (_, { elementType }) => getSchemaAttributeValues(elementType),
+    schemaAttributeNames: (_, { elementType }) => getSchemaAttributeNames(elementType),
```

Comment:
Just curious, why change `schemaAttributes` to `schemaAttributeNames` ?

Reply:
Because, we don't return a list of attributes but we return a list of names attributes.
the method now better represents what it does